db/fixtures/miq_report_formats.yml

```
@@ -649,6 +649,14 @@
       :name: datetime
       :format: "%m/%d"
 
+  :relative_date:
+    :description: Relative Date
+    :data_types:
+    - :date
+    - :datetime
```

I think we can also support timestamp here.
```suggestion
    - :datetime
    - :timestamp
```

---

used **datetime and timestamp** as the datatypes and removed **date** datatype